babel/solcWorker2.js

```diff
@@ -0,0 +1,42 @@
+import wrapper from 'solc/wrapper'
+
+let loadedCompilers = {}
+
+function loadCompiler(version) {
+  if (!loadedCompilers[version]) {
+    const url = `https://binaries.soliditylang.org/bin/soljson-${version}.js`
+    try {
+      // eslint-disable-next-line no-undef
+      importScripts(url)
+    } catch (e) {
+      return false
+    }
+    loadedCompilers[version] = true
+  }
+
+  return true
+}
+
+function onCompileRequest(msg) {
+  let { version, stdJson } = msg.data
+  if (!version || !stdJson) {
+    self.postMessage({ error: 'no version or standard json specified' })
+    return
+  }
+
+  // console.log('loading compiler:', url)

```

Comment:
Remove all such console.log comments across all changed files when you're ready for a final review & have finished developing locally.